TypeCobol.Test/Parser/Programs/Cobol85/MissingSpaceAfterDotDecimalPointIsComma.rdz.cbl

```diff
@@ -0,0 +1,57 @@
+﻿       IDENTIFICATION DIVISION.
+      *Warning : A blank was missing before character "P" in column 19.
+      *          A blank was assumed.
+       PROGRAM-ID.Pgm.
+      * OK: No warning
+       AUTHOR.AUTHOR.
+      * OK: No warning
+       DATE-WRITTEN.NOVEMBRE 80.
+      * OK: No warning
+       DATE-COMPILED..
+       ENVIRONMENT DIVISION.
+       CONFIGURATION SECTION.
+      *Warning: A blank was missing before character "I" in column 24.
+      *         A blank was assumed.
+       SOURCE-COMPUTER.IBM-370.
+      *Warning: A blank was missing before character "I" in column 24.
+      *         A blank was assumed.
+       OBJECT-COMPUTER.IBMCOBOLII.
+      *Warning: A blank was missing before character "D" in column 22.
+      *         A blank was assumed.
+       SPECIAL-NAMES.DECIMAL-POINT IS COMMA.
```

Comment:
We have 2 almost identical tests to ensure that `SPECIAL-NAMES.DECIMAL-POINT IS COMMA.` doesn't change the result.

Reply:
True, as `SPECIAL-NAMES.DECIMAL-POINT IS COMMA.` modifies the behaviour of the period character and the paths in the code are a bit different, it might be interresting to keep this test.